pkg/controller/v1beta1/inferenceservice/reconcilers/keda/keda_reconciler.go

```diff
@@ -215,6 +244,25 @@ func createKedaScaledObject(componentMeta metav1.ObjectMeta,
 		},
 	}
 
+	if scaleDownStabilizationWindowSeconds != nil || scaleUpStabilizationWindowSeconds != nil {
+		hpaBehavior := &autoscalingv2.HorizontalPodAutoscalerBehavior{}
```

Comment:
Can we initialize with `componentExtension.AutoScaling.Behavior` and then override the ScaleDown, ScaleUp, otherwise what user specified on `componentExtension.AutoScaling.Behavior` is not taking effect in KEDA.